ui/partials/main/thActiveFiltersBar.html

```diff
@@ -1,20 +1,27 @@
 <!-- Show this certain filters are active in Treeherder -->
-<div class="alert-info active-filters-bar">
+<div ng-if="newFieldFilter || filterBarFilters.length"
+     class="alert-info active-filters-bar">
     <div ng-show="filterBarFilters.length" ng-cloak>
-      <i class="fa fa-info-circle" aria-hidden="true"></i>
-      <span><b>Active Filters</b></span>
-
-      <span ng-repeat="filter in filterBarFilters" class="btn btn-sm"
-            title="Click to clear the {{ filter.field }} filter"
-            ng-click="jobFilters.removeFilter(filter.key, filter.value)">
-          <span ng-if="filter.field == 'author'">{{ filter.field }}: {{filter.value.split('@')[0] | limitTo: 20}}</span>
-          <span ng-if="filter !== 'author'">{{ filter.field }}: {{filter.value | limitTo: 12}}</span>
-          <i class="fa fa-times-circle"></i>
+      <span class="active-filters-title">
+        <i class="fa fa-info-circle" aria-hidden="true"></i>
+        <b>Active Filters</b>
+      </span>
+      <span ng-repeat="filter in filterBarFilters"
+            class="filtersbar-filter">
+          <span title="Filter by {{ filter.field}}: {{ filter.value }}"
```

Comment:
I could probably write some really long ternary here to save replicating the title property twice. But I didn't for the initial review.